client/state/plugins/premium/selectors.js

```diff
@@ -92,3 +98,21 @@ export const getNextPlugin = function( state, siteId, whitelist = false ) {
 	}
 	return plugin;
 };
+
+export const getPluginKeys = createSelector(
+	( state, siteId, whitelist = false ) => {
+		const pluginList = getPluginsForSite( state, siteId, whitelist );
+
+		return pluginList.reduce( ( keys, plugin ) => {
+			const key = get( plugin, 'key', null );
+			const slug = get( plugin, 'slug', null );
+
+			if ( key && slug ) {
+				keys[ slug ] = key;
```

Comment:
Nit, I think we could avoid mutating `keys` here by doing

```js
return {
	...keys,
	[ slug ]: key
};
```

here.

Reply:
Nice find! Fixed in be76b80ef7198dcf6a7707e7d3cccaea82c0bf4b :+1: